lunes_cms/cms/models/document.py

```diff
@@ -62,6 +62,7 @@ class Document(models.Model):
     creator_is_admin = models.BooleanField(default=True, verbose_name=_("admin"))
     feedback = GenericRelation(Feedback)
 
+    # pylint: disable=consider-using-with
```

Comment:
```suggestion
```
I think in this case it would be better to actually use `with` incited of ignoring the warning.

Reply:
Yes I tried, the problem was `seek of closed file` error, becuase the opened file has to stay opened while the returned value is being read in the save function.

Shall we move the code of `converted` into the function `save`?

Reply:
Hmm ok, then leave the `disable` for the moment, maybe I will have a look in ~2 weeks again :sweat_smile: